android/src/androidTest/java/co/nimblehq/ic/kmm/suv/android/ui/screens/login/LoginScreenTest.kt

```diff
@@ -1,29 +1,35 @@
 package co.nimblehq.ic.kmm.suv.android.ui.screens.login
 
 import androidx.activity.compose.setContent
-import androidx.compose.ui.test.ExperimentalTestApi
-import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.*
 import androidx.compose.ui.test.junit4.createAndroidComposeRule
-import androidx.compose.ui.test.onNodeWithText
 import co.nimblehq.ic.kmm.suv.android.MainActivity
 import co.nimblehq.ic.kmm.suv.android.R
+import co.nimblehq.ic.kmm.suv.domain.model.AppError
 import co.nimblehq.ic.kmm.suv.domain.usecase.LogInUseCase
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.flowOf
+import org.junit.Assert.assertTrue
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
-import org.koin.java.KoinJavaComponent
 
 @ExperimentalTestApi
 class LoginScreenTest {
 
     @get:Rule
     val composeRule = createAndroidComposeRule<MainActivity>()
 
+    private lateinit var viewModel: LoginViewModel
+    private val mockLogInUseCase = mockk<LogInUseCase>()
```

Comment:
JIKYDK, we could define the mock UseCase like this: `private val mockLogInUseCase: LogInUseCase = mockk()`, the it would align with the ViewModel definition.

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/92/commits/c3e5eee44a768ed400b77ebb78d1ae4ab69c1567